code/game/machinery/computer/dropship_weapons.dm

```diff
@@ -589,8 +589,9 @@
 
 	ui_interact(usr)
 
-/obj/structure/machinery/computer/dropship_weapons/proc/remove_from_view()
+/obj/structure/machinery/computer/dropship_weapons/proc/remove_from_view(var/mob/living/carbon/human/usr)
```

Comment:
name it `user`, not `usr`, also no `var/` in args

Reply:
Do I just rename usr in the args or everywhere in the proc? I assume proc. Just claryfing, so I dont mess up things more

Reply:
Rename it all to user

user argument > the inherent byond usr

Reply:
so for whatever reason the unregister signal only works with usr (probably because its registers with usr) otherwise the thing stops and tells it reset view was overriden

Reply:
other then that its done

Reply:
fuick

Reply:
by "rename it all"
I guess you meant the whole file, oh well.